src/components/MoneyRequestHeader.tsx

```diff
@@ -181,15 +185,29 @@ function MoneyRequestHeader({session, parentReport, report, parentReportAction,
                 />
                 {isPending && (
                     <MoneyRequestHeaderStatusBar
-                        title={translate('iou.pending')}
-                        description={translate('iou.transactionPendingText')}
+                        title={
+                            <Icon
+                                src={Expensicons.CreditCardHourglass}
+                                height={variables.iconSizeSmall}
+                                width={variables.iconSizeSmall}
+                                fill={theme.textSupporting}
```

Comment:
I think this color is wrong, see @dannymcclain's guide here:
![image](https://github.com/Expensify/App/assets/2319350/53f82921-1dda-42d0-831a-ff8132dc36b0)

The color should be color.icons

Reply:
Can you also show us where you are setting the gap between the icon and the text?

Reply:
`MoneyRequestHeaderStatusBar`:
`<View style={styles.mr2}>{title}</View>`

I cannot generalize it because it's also used in `SplitBillDetails`